app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -25,6 +25,14 @@
   <dimen name="profile_chooser_profile_view_margin_start_profile_not_added">24dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_end_profile_not_added">24dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_top_profile_not_added">24dp</dimen>
+
+  <!-- Profile Chooser Fragment -->
+  <dimen name="profile_chooser_fragment_layout_margin_start">96dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_bottom">28dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_end">96dp</dimen>
+  <dimen name="profile_chooser_fragment_margin_top_profile_already_added">0dp</dimen>
+  <dimen name="profile_chooser_fragment_margin_top_profile_not_added">84dp</dimen>
+
```

Comment:
If the `profile_chooser_fragment_margin_top_profile_already_added` and `profile_chooser_fragment_margin_top_profile_not_added` are already defined in the portrait xml (mobile). Then we do not need to denfine it here again.